spring-cloud-gcp-autoconfigure/src/test/java/com/google/cloud/spring/autoconfigure/sql/CloudSqlEnvironmentPostProcessorTests.java

```diff
@@ -114,7 +114,7 @@ public void testCloudSqlDataSourceWithIgnoredProvidedUrl() {
 	}
 
 	@Test
-	public void testCloudSqlAppEngineDataSourceDefaultUserNameRootTest() {
+	public void testCloudSqlAppEngineDataSourceDefaultUserNameMySqlTest() {
```

Comment:
This is a Postgres test, right? `FilteredClassLoader` removes classes, so mysql is removed here.

Reply:
Where is `FilteredClassLoader` in this test?

Reply:
Never mind, github collapsed this visually with the next change, and I did not notice.